src/esp/assets/ResourceManager.cpp

```diff
@@ -333,11 +333,63 @@ int ResourceManager::loadObject(const std::string objPhysConfigFilename)
   //2. construct a physicsObjectMetaData
   PhysicsObjectMetaData physMetaData;
 
-  //3. load/check_for render and collision mesh metadata
-  //    ALEX NOTE: these paths should be relative to the properties file
+  //ALEX NOTE: these paths should be relative to the properties file
   std::string propertiesFileDirectory = objPhysConfigFilename.substr(0,
       objPhysConfigFilename.find_last_of("/"));
```

Comment:
[Utility::Directory::path()](https://doc.magnum.graphics/corrade/namespaceCorrade_1_1Utility_1_1Directory.html#a222baead7128ec0cadc433e3a8c0059f), there's also a lot more utilities for path handling